<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>algorithm_page</class>
 <widget class="QWidget" name="algorithm_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>560</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>algorithm_page</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>631</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>AStar</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>403</width>
       <height>387</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="title">
      <string>AStar parameter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Min Launch Height</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="lineEdit_20"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Step Length</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_21"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Survivability Weight</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Max Launch Height</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="lineEdit_19"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Target Weight</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLineEdit" name="lineEdit_23">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Start Weight</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Verticality Corner1</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLineEdit" name="lineEdit_22"/>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Horizontal Corner</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>E Weight</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>DE</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>411</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>DE parameter</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>242</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Initial Vertex Number</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Evolution Number</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Weight Factor</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Population Number</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Cross Probability</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="text">
          <string>200</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="text">
          <string>20</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_8">
         <property name="text">
          <string>1000</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_9">
         <property name="text">
          <string>0.5</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_10">
         <property name="text">
          <string>0.9</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Pso</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>30</y>
       <width>451</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>PSO Parameter</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>363</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_50"/>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEdit_51"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_50">
         <property name="text">
          <string>Swarm Size</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_51">
         <property name="text">
          <string>Loop Couner</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>591</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Target Function Weight Values</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>538</width>
        <height>95</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="2">
        <widget class="QLabel" name="label_52">
         <property name="text">
          <string>OilCost</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEdit_52"/>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_53">
         <property name="text">
          <string>Missions</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_54">
         <property name="text">
          <string>HightConstrain</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEdit_53"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_54"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_55">
         <property name="text">
          <string>SurvivalCost</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_55"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_56">
         <property name="text">
          <string>RefPath</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="lineEdit_56"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>451</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Dynamic Search Strategy Parameter</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>377</width>
        <height>95</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_57"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_57">
         <property name="text">
          <string>Eecute Step</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_58">
         <property name="text">
          <string>Step Distance</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEdit_58"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_59">
         <property name="text">
          <string>Search Step</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_60">
         <property name="text">
          <string>Pitch(0~45)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_59"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_60"/>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEdit_61"/>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEdit_62"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_61">
         <property name="text">
          <string>Detect Range</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_62">
         <property name="text">
          <string>Yaw(0~30)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>669</x>
     <y>40</y>
     <width>271</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Transition intensity in Radar area</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>234</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="text">
        <string>0.3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Lud</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="text">
        <string>0.2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_14">
       <property name="text">
        <string>0.2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_17">
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_15">
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_16">
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Ldu</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Ldt</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Ltd</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Lte</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Let</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Leh</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>510</y>
     <width>203</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>351</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>OwnPlatform:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>MissionType:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Route Name:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_18">
      <property name="text">
       <string>Route1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>algorithm_page</receiver>
   <slot>on_datapage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>826</x>
     <y>642</y>
    </hint>
    <hint type="destinationlabel">
     <x>1287</x>
     <y>629</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_datapage()</slot>
 </slots>
</ui>
